src/views/shortcodes/my-attendance-list.php

```diff
@@ -6,22 +6,41 @@
  *
  *     [your-theme]/tribe-events/shortcodes/my-attendance-list.php
  *
- * @version 4.8.2
+ * @since   4.8.2
+ * @since   TBD Removed target="_blank" from links, added direct link to each post's "My Tickets" view.
+ *
+ * @version TBD
  *
  * @var array $event_ids
  */
+
+$view = Tribe__Tickets__Tickets_View::instance();
 ?>
 
 <ul class="tribe-tickets my-attendance-list">
-	<?php foreach ( $event_ids as $id ): ?>
+	<?php
+	foreach ( $event_ids as $id ):
```

Comment:
* :no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $id
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and T_COLON is required

<sub>hash: 82c030ec7db1eb360fe5349fc83be2d0</sub>

Reply:
```suggestion
	foreach ( $event_ids as $id ) :
```